controller/setting_controller.go

```diff
@@ -1371,11 +1379,15 @@ func (info *ClusterInfo) collectSettings() error {
 
 		// Setting that should be collected as boolean (true if configured, false if not)
 		case includeAsBoolean[settingName]:
-			settingMap[setting.Name] = fmt.Sprint(setting.Value != "")
+			settingMap[setting.Name] = setting.Value != ""
 
 		// Setting value
 		case include[settingName]:
-			settingMap[setting.Name] = setting.Value
+			convertedValue, err := info.convertSettingValueType(setting)
+			if err != nil {
+				logrus.WithError(err).Debugf("failed to convert Setting %v value", setting.Name)
+			}
+			settingMap[setting.Name] = convertedValue
```

Comment:
if failed to convert, do we still need to collect that value? probably just exclude to prevent any bias or potential error?